libreofficeid/LO-TC-GUI
# Add Import functionality

how to test ?
- create lotc file (actually, it is zip file)
- open the extension
- click `Import Theme` button, then import dialog will appear
- locate your lotc file using `browse` button, then
- click `Import ` button to proceed

result:
- your theme name will be listed on main dialog
- your theme will be live on `$HOME/.config/libreoffice/4/user/lotc-themes/`

~TODO:~
~- open details dialog on double clicking theme name in theme list~
